src/scripts/manually_filter_reddit_comments.py

```diff
@@ -0,0 +1,140 @@
+"""Script for manually filtering a list of reddit comments."""
+
+import pandas as pd
+from datasets import Dataset, load_dataset
+import nltk
+import click
+from pathlib import Path
+from nltk.tokenize import sent_tokenize
+import itertools as it
+import re
+
+
+# Download the sentence splitter model
+nltk.download("punkt", quiet=True)
+
+
+@click.command("Starts the process of manually filter reddit comments.")
+@click.option("--output-dir", type=Path, required=True)
+@click.option("--n", type=int, default=1000)
+@click.option("--start-index", type=int, default=0)
+@click.option("--username", type=str, required=True)
+def filter_reddit_dataset(
+    output_dir: Path | str, username: str, n: int = 1000, start_index: int = 0
+):
+    """Script for manually filtering a list of reddit comments.
+
+    Args:
+        output_dir: The directory to save the dataset to.
+        username: The username of the person filtering the dataset.
+        n: The number of sentences to manually filter.
+        start_index: The index to start from.
+    """
```

Comment:
This is not needed when using `click`:
```suggestion
def filter_reddit_dataset(output_dir, username, n, start_index):
    """Script for manually filtering a list of reddit comments."""
```